Headers/AppKit/NSUserActivity.h

```diff
@@ -0,0 +1,47 @@
+/* Definition of class NSUserActivity
```

Comment:
Why is this file included in this PR? It doesn't do any harm but most likely it was not planed to be here.